components/dashboard/src/App.tsx

```diff
@@ -46,7 +46,7 @@ function App() {
             }
             setLoading(false);
         })();
-    }, []);
+    }, [ setUser ]);
```

Comment:
@svenefftinge we need to change this :-(

this effect causes constant stream of `getLoggedInUser` requests 

<img width="1070" alt="Screen Shot 2021-04-01 at 05 52 56" src="https://user-images.githubusercontent.com/914497/113241261-9f545100-92ae-11eb-8d5c-e9bba354a29e.png">